vendor/parser-extended/parser.mly

```diff
@@ -1357,12 +1361,6 @@ structure_item:
         { pstr_typext $1 }
     | str_exception_declaration
         { pstr_exception $1 }
-    | module_binding
-        { $1 }
-    | rec_module_bindings
-        { pstr_recmodule $1 }
-    | module_type_declaration
-        { let (body, ext) = $1 in (Pstr_modtype body, ext) }
```

Comment:
I prefer code to be commented-out than removed in the parser. This greatly helps backporting upstream changes.
If possible with a very short explanation.

Reply:
It is not removed but moved outside of the wrap_mkstr_ext. When this is done, maybe the diff will more legible because wrap_mkstr_ext will not exist anymore

Reply:
Fine to me.